src/python_framework_v02/troute/network/reservoirs/hybrid/hybrid_structs.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HYBRID_STRUCTS_H
+#define HYBRID_STRUCTS_H
+/*
+    C Structures
+*/
+#include "../../reach_structs.h"
+
+#define MAX_PATH_LENGTH 256

```

Comment:
@hellkite500 Could you verify these maxes per our conversation from the other day? The start date should be 19 char.